invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -171,6 +234,10 @@ class UIRecordSchema(BaseObjectSchema):
         partial(CustomFieldsSchemaUI, fields_var="RDM_CUSTOM_FIELDS")
     )
 
+    publishing_information = fields.Function(compute_publishing_information)
+
+    conference = fields.Function(compute_conference_information)
```

Comment:
Comment: Maybe I am saying something stupid here, but couldn't this be replaced by:
```suggestion
    conference = fields.Nested(<your_field_class>, attribute="custom_fields.meeting:meeting")
```